testing/unittests/metrics/MetricFrameworkTests.cpp

```diff
@@ -418,6 +429,15 @@ class MetricFrameworkTests : public CppUnit::TestFixture
             CPPUNIT_ASSERT_EQUAL(static_cast<int>(pHistogram->queryValue()), expectedSum);
     }
 
+
+    void checkHistogramValue(std::shared_ptr<IMetric> pHistogram, __uint64 expectedValue, __uint64 error)
```

Comment:
trivial: Slightly better if 1st parameter is a reference (but doesn't really matter since test code).